### PR TITLE
Sri/jointeam

### PR DESCRIPTION
**Pull request by sri240**
Created "My Team" page for when users are not in a team, allowing for team creation and joining a team.

### CODE REVIEW

**Review by lowtorola — CHANGES_REQUESTED** [review]
Looks great! Should be good to re-request for merging after the few nits I commented. Also, can you send a screenshot of what the page looks like to the PR (just for my own curiosity)? 🎉

File: frontend2/src/components/team/EligibilitySettings.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const EligibilitySettings: React.FC = () => {
     !isPresent(team) ||
     !isPresent(episode)
   ) {
-    return "Error: You're not in a team!";
+    return <div></div>;
```
Comment (acrantel) [inline frontend2/src/components/team/EligibilitySettings.tsx:79]: We can place the string `Error: You're not in a team!` inside the div here to display it

**Review by acrantel — COMMENTED** [review]
great work! as Lowell said, you should run `npm run format` in the frontend2 folder to format everything nicely

**Comment by sri240** [comment]
Screenshot 2023-12-02 at 2 29 49 PM

**Comment by acrantel** [comment]
lgtm! nice work 🥳

**Review by acrantel — APPROVED** [review]

**Pull request merged**